app/preprints/-components/preprint-institutions/institution-manager/component.ts

```diff
@@ -85,4 +87,8 @@ export default class InstitutionsManagerComponent extends Component<InstitutionA
     toggleInstitution(institution: PreprintInstitutionModel) {
         this.manager.updateAffiliatedInstitution(institution);
     }
+
+    public get disableWriteAccess(): boolean {
```

Comment:
Slight nit with this getter having a verb name. Could be something like `currentUserIsAdmin` but then some of the logic would need to get flipped.